### DISCUSSION TITLE
Solcast free trial plan ending?

### DISCUSSION THREAD

**Discussion by kme77** [nathanmarlor/foxess_modbus#587] [Q&A]
Has  anyone else received an email saying the Solcast free tier will be moving to a paid model from 16th April? If the cost is prohibitive (i.e. costs more than we would save using it) is there an alternative or would the solar forecast portion of the integration need to be removed/deactivated?

**Comment by canton7** [comment]
I haven't (yet?). Note that this integration does not use solar forecasts at all, and knows nothing about solcast - you might be thinking of foxess_em.

**Reply by kme77** [reply]
Ahh.. I've posted in the wrong forum (again!) I'll repost in the correct one, cheers!
